Remove cutQC (#605)
Qiskit-Extensions/circuit-knitting-toolbox: INSTALL.rst

Platform Support
^^^^^^^^^^^^^^^^

> We expect this package to work on any common OS and CPU architecture. If

Reviewer: ```suggestion
We expect this package to work on `any platform supported by Qiskit <https://docs.quantum.ibm.com/start/install#operating-system-support>`__. If
```

you are experiencing issues running the software on your device, you

> may consider :ref:`using the toolbox within Docker <Option 3>`.

Reviewer: IMO the file should end with this line. We no longer need the bullet point after this -- about installing an x86 version of python or using conda -- now that pyscf and cplex are both no longer required.
